blocks/feeds-source-video-api-block/package.json

```diff
@@ -1,26 +1,22 @@
 {
   "name": "@wpmedia/feeds-source-video-api-block",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "description": "Fusion components for building sitemaps",
   "main": "index.js",
   "files": [
     "sources"
   ],
-  "license": "UNLICENSED",
+  "license": "wapo/arc",
```

Comment:
is this a real license?

Reply:
No, But that is what Hardip told me to use.  Looking at the ANS Schema which is public they used MIT.